app/controllers/categories_controller.rb

```diff
@@ -12,11 +12,17 @@ def subcategories
   end
 
   def index
-    categories = Category.order(:name)
+    # categories = Category.order(:name)
     # Cast:
     #   nil and '' -> nil
     #   '0', 'false', 'False', 'f', etc. -> false
     #   Almost everything else -> true
+    categories = if params[:site_id]
+                   Site.find(params[:site_id]).categories
+                 else
+                   Site.find(0).categories #sfsg
```

Comment:
`sfsg` is not necessarily site_id 0 - a safer way would be using `find_by(site_code: 'sfsg')`.